sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/DetectLanguageInput.java

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.models;
+
+import com.azure.core.annotation.Immutable;
+
+/**
+ * The DetectLanguageInput model.
+ */
+@Immutable
+public final class DetectLanguageInput {
+    /*
+     * Unique, non-empty document identifier.
+     */
+    private final String id;
```

Comment:
Does the service return IDs as Strings? Every use of setting the property is `Integer.toString` from what I have seen.

Reply:
ID can be anything. Not just integers

Reply:
ID as String